spec/spec_helper_acceptance.rb

```diff
@@ -45,13 +45,13 @@ def expect_no_change_from_manifest(manifest)
 end
 
 def http_package_url
-  url_root = 'http://download.elasticsearch.org/logstash/logstash/packages'
+  url_root = 'https://download.elastic.co/logstash/logstash/packages'
 
   case fact('osfamily')
   when 'Debian'
-    "#{url_root}/debian/logstash_#{LS_VERSION}-1_all.deb"
+    "#{url_root}/debian/logstash-#{LS_VERSION}_all.deb"
   when 'RedHat', 'Suse'
-    "#{url_root}/centos/logstash-#{LS_VERSION}-1.noarch.rpm"
+    "#{url_root}/centos/logstash-#{LS_VERSION}.noarch.rpm"
```

Comment:
The same `-1` change applies here too. It seems we are going to need some conditional logic to deal with the change in naming convention.